sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseService.java

```diff
@@ -1110,6 +1114,12 @@ private void deleteCaseLinks(Case caze) {
 
 		// Remove the case as the related case of immunizations
 		immunizationService.unlinkRelatedCase(caze);
+
+		// Remove the case from any external message referencing it
+		externalMessageService.getForCase(new CaseReferenceDto(caze.getUuid())).forEach(labMessage -> {
```

Comment:
There is a small naming issue with this. 
SORMAS has external messages now, which can be of type lab message or physician's report. Actually, external messages with a case reference are always phisycan's reports, never lab messages. Thus, the parameter name labMessage is misleading here. @MateStrysewske Shall I just push a small fix of this (tagging #10050) to development?